src/extensions/Markdown.js

```diff
@@ -79,6 +79,7 @@ const createMarkdownSerializer = ({ nodes, marks }) => {
 		),
 		serialize(content, options) {
 			return this.serializer.serialize(content, { ...options, tightLists: true })
+				.split('\\\n').join('  \n')
```

Comment:
what will happen to a line that ends with a literal `\` in the editor?
I suspect it will be transformed into a line ending with `\\` by the serialization and then turned into `\  ` as the end by this regexp.

Maybe better overwrite the `to_markdown` property of the `HardBreak` node?

Reply:
https://github.com/ProseMirror/prosemirror-markdown/blob/master/src/to_markdown.ts#L107-L113 could serve as a starting point.

Reply:
Move to an extension that implements toMarkdown and added test cases